tpl/html.tpl.php

```diff
@@ -24,5 +25,8 @@
   <?php print $page_top; ?>
   <?php print $page; ?>
   <?php print $page_bottom; ?>
+
+  <?php echo file_get_contents( drupal_get_path('theme',$GLOBALS['theme']) . '/img/icons/icons-sprite.svg'); ?>
```

Comment:
I think we could just hardcode to `ocha_basic` here since that's where the sprite is. For example, what if someone did try to use this as a base theme and their active theme is called `hdx` or whatever? I think this global would point to the active theme dir.

Reply:
Also this is very slick in that it also inlines our SVGs without spreading them all over the page ⚡️